pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -107,7 +107,8 @@ def advance(  # type: ignore[override]
         void(dl_max_batches)
 
         assert self._dataloader_iter is not None
-        with self.trainer.profiler.profile("{}_dataloader_next"):
+        action_name = "test_dataloader_next" if self.trainer.testing else "val_dataloader_next"
```

Comment:
let's follow the action name style here so it's uniform in the profiler later
https://github.com/PyTorchLightning/pytorch-lightning/blob/7e2f9fbad555242b0ceb2a24e5e4c004f0701bae/pytorch_lightning/trainer/trainer.py#L1604

        with self.profiler.profile(f"[LightningModule]{pl_module.__class__.__name__}.{hook_name}"):

```suggestion
        suffix = "test_dataloader_next" if self.trainer.testing else "val_dataloader_next"
        action_name = f"[{self.__class__.__name__}].{suffix}"
```

Reply:
There are a couple spots that don't follow this convention. Should these be updated too?

- https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/loops/fit_loop.py#L273
- https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/loops/epoch/training_epoch_loop.py#L198